templates/workspaces/base/terraform/network/network.tf

```diff
@@ -187,6 +193,22 @@ resource "azurerm_network_security_rule" "allow-inbound-from-bastion" {
   source_port_range = "*"
 }
 
+resource "azurerm_network_security_rule" "allow-inbound-from-resourceprocessor" {
+  access                       = "Allow"
+  destination_address_prefixes = azurerm_subnet.services.address_prefixes
+  destination_port_range       = "*"
```

Comment:
Do we need all ports? Is it just 443?

Reply:
443 should probably be enough; I'm at least not aware of any other ports that might be needed. I'll adjust.

Reply:
Done